### PR TITLE
Update getting started guide to 0.3

### PR DESCRIPTION
**Pull request by msujew**
Includes minor changes from 0.3 which have to be reflected in the documentation.

### CODE REVIEW

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -72,12 +72,12 @@ For a full explanation of all terms we use throughout Langium, please refer to o
 Here's the grammar that parses the previous text snippet:
 
 ```
```
Comment (gfontorbe) [inline content/docs/getting-started.md:74]: The grammar that is generated by `yo langium` is actually a bit different, terminals are at the end, there are terminal rules for INT STRING and comments, and type for the reference to Person:
```
grammar HelloWorld

entry Model:
    (persons+=Person | greetings+=Greeting)*;

Person:
    'person' name=ID;

Greeting:
    'Hello' person=[Person:ID] '!';

hidden terminal WS: /\s+/;
terminal ID: /[_a-zA-Z][\w_]*/;
terminal INT returns number: /[0-9]+/;
terminal STRING: /"[^"]*"|'[^']*'/;

hidden terminal ML_COMMENT: /\/\*[\s\S]*?\*\//;
hidden terminal SL_COMMENT: /\/\/[^\n\r]*/;
```

**Review by gfontorbe — COMMENTED** [review]
The grammar file is a bit different on a newly generated project. Apart from that, LGTM.

File: content/docs/getting-started.md (hunk 1 above)
Comment (msujew) [reply content/docs/getting-started.md:74]: Given that these rules are basically only included to have reference points how these terminals would usually look like, I would not try to include them in the getting started page. The point of this page is to explain how one would get started with a Langium project. Throwing unused terminals in there likely just leads to confusion. @spoenemann WDYT?

Comment (spoenemann) [reply content/docs/getting-started.md:74]: Yes, let's omit unused terminals to avoid confusion.

**Review by spoenemann — APPROVED** [review]

**Pull request merged**